seed/lib/xml_mapping/mapper.py

```diff
@@ -56,20 +56,24 @@ def get_bae_mappings():
     # units field name is the same with " Units" appended.
 
     bsync_assets = BAE.get_default_asset_defs()
-    for item in bsync_assets:
+    for asset in bsync_assets:
+        if isinstance(asset, dict):
```

Comment:
This should never happen with the latest BAE code. @axelstudios  - you probably just needed to run `pip install -r requirements/local.txt`